### ISSUE TITLE
qmpのnode nameが長すぎると切り捨てられるためdevice addで参照できずエラーが起きる

### ISSUE THREAD

**Issue by h-otter** [n0stack/n0stack#144]
```
---> [ 10/12 ] Task 'CreateApplicationServerVirtualMachine' is failed: got error response: rpc error: code = Internal desc = Failed to BootVirtualMachineRequest: rpc error: code = Internal desc = Failed to CreateVirtualMachineAgent: desc=Failed to attach image 'file:///var/lib/n0core/block_storage/****': err='Failed to create virtio block device: err='Property 'virtio-blk-device.drive' can't find value '****'''
```

node nameが39文字だと駄目
18文字はOK

**Comment by h-otter** [comment]
>node-name: string (optional)
>the node name of the new node (Since 2.0). This option is required on the top level of blockdev-add. Valid node names start with an alphabetic character and may contain only alphanumeric characters, ’-’, ’.’ and ’_’. Their maximum length is 31 characters.

https://qemu.weilnetz.de/doc/qemu-qmp-ref.html

**Comment by h-otter** [comment]
netdevみたいに4文字のcrc checksumをつけると良さそう

**Comment by h-otter** [comment]
fixed on 524fcacdaeca5de5bf74d07b8b58c4002cac55f1
